model/riscv_sys_regs.sail

```diff
@@ -180,6 +180,22 @@ function haveUsrMode() -> bool = misa.U() == 0b1
 function haveNExt()    -> bool = misa.N() == 0b1
 /* see below for F and D extension tests */
 
+/* Cryptography extension support. Note these will need updating once */
+/* Sail can be dynamically configured with different extension support */
+/* and have dynamic changes of XLEN via S/UXL */
+function haveZkr()    -> bool = true
+function haveZksh()   -> bool = true
+function haveZksed()  -> bool = true
+function haveZknh()   -> bool = true
+function haveZknh32() -> bool = haveZknh() & sizeof(xlen) == 32
```

Comment:
These aren't real extension names, and if they were would mean version 32.0 of the Zknh extension; please just inline these as is done for the various other XLEN-specific instructions

Reply:
Sure thing, I've done this in the latest commits.